target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -34,6 +34,13 @@ gnubee,gb-pc2)
 	ucidef_set_led_netdev "lan1" "lan1" "$boardname:green:lan1" "lan1"
 	ucidef_set_led_netdev "lan2" "lan2" "$boardname:green:lan2" "lan2"
 	;;
+linksys,ea7500-v2)
+	ucidef_set_led_netdev "lan1" "lan1 link" "$boardname:green:lan1" "lan1" "link"
+	ucidef_set_led_netdev "lan2" "lan2 link" "$boardname:green:lan2" "lan2" "link"
+	ucidef_set_led_netdev "lan3" "lan3 link" "$boardname:green:lan3" "lan3" "link"
+	ucidef_set_led_netdev "lan4" "lan4 link" "$boardname:green:lan4" "lan4" "link"
+	ucidef_set_led_netdev "wan" "wan link" "$boardname:green:wan" "wan" "link"
```

Comment:
Link is default and can be dropped (like for all the other devices :-) ).

Reply:
The the router uses LAN ports with orange and green LEDS (similar to common PCs https://avtech.com/articles/wp-content/uploads/2015/06/Ethernet_LEDs_No_Power.gif).
Green LEDs at the back of the router indicates connectivity and should not blink when data is transmitted

The orange LEDs that blinks then data is transmitted seems to be controlled by the hardware directly (not connected to GPIO), so they don't need configurations.

Reply:
okay, I think most people would have just set the green leds to blink as well, but +1 for reproducing vendor behavior here :-)